#112: Fixed Parquet repeated reader bugs (#121)
exasol/cloud-storage-extension: doc/user_guide/user_guide.md

| group                | MAP                   | VARCHAR(n)                      |
| group                | REPEATED              | VARCHAR(n)                      |

#### Parquet Repeated Types

Parquet data type can repeated on a single field or on the group of fields.
Repeated fields and repeated groups with a single field are imported as JSON
array string.

> For example, the following two schemas are imported as JSON array strings.

Reviewer: passive

```
message parquet_schema {
  repeated binary name (UTF8);
}
```

```
message parquet_schema {
  repeated group person {
    required binary name (UTF8);
  }
}
```

> Both of these Parquet types are imported as a JSON array `["John","Jane"]`.

Reviewer: ```suggestion
The Parquet importer imports both of these Parquet as a JSON array `["John","Jane"]`.
```

> On the other hand, a repeated group with multiple fields are imported as JSON

Reviewer: passive
